### DISCUSSION TITLE
Integrate on scaled data or not?

### DISCUSSION THREAD

**Discussion by Jeff-Gui** [brianhie/scanorama#97] [Q&A]
Hi, I wonder what pre-processing I must do before integration. Shall I do logNorm only or additionally scale data? I cannot clearly recognize this from the given tutorials.

**Comment by brianhie** [comment]
Hi @Jeff-Gui, in this benchmarking preprint here: https://www.biorxiv.org/content/10.1101/2020.05.22.111161v2.full, Scanorama performs best restricted to highly variable genes and with additional data scaling. Hope that helps!
